applications/GeoMechanicsApplication/custom_python/geo_mechanics_python_application.cpp

```diff
@@ -52,9 +52,12 @@ PYBIND11_MODULE(KratosGeoMechanicsApplication,m)
     //Registering variables in python
     KRATOS_REGISTER_IN_PYTHON_VARIABLE( m, VELOCITY_COEFFICIENT )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE( m, DT_PRESSURE_COEFFICIENT )
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE( m, DT_TEMPERATURE_COEFFICIENT )
```

Comment:
Please check the spaces in the beginning of the line

Reply:
Done